package.json

```diff
@@ -96,11 +74,10 @@
       "^@/(.*)$": "<rootDir>/$1"
     },
     "transform": {
-      "^.+\\.js$": "babel-jest",
```

Comment:
what if someone wants to have js test files as well?

Reply:
@husayt Best setup for handling js files would be to tell `ts-jest` to handle them with `allowJs: true` in `tsconfig.json` + change the regex to match `.js` test files